_studio/mfx_lib/fei/h264_la/mfx_h264_la.cpp

```diff
@@ -483,6 +481,33 @@ mfxStatus VideoENC_LA::Init(mfxVideoParam *par)
     mfxFrameAllocRequest request = { };
     request.Info = m_video.mfx.FrameInfo;
 
+    if (m_video.IOPattern == MFX_IOPATTERN_IN_SYSTEM_MEMORY)
```

Comment:
m_raw used for copy surfaces to video memory in case of system input.
If DS factor > 1, then we alloc pool with wrong size, because we already
changed request.